client/my-sites/stats/activity-log/activity-log-success-banner.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActivityLogBanner from './activity-log-banner';
+import Button from 'components/button';
+
+function ActivityLogSuccessBanner( {
+	translate,
+	moment,
+} ) {
+	// FIXME: real dates
+	const date = 1496400468285;
+	// FIXME: real dismiss
+	const handleDismiss = () => {};
+
+	return (
+		<ActivityLogBanner
+			isDismissable
+			onDismissClick={ handleDismiss }
+			status="is-success"
+			title={ translate( 'Your site has been successfully restored' ) }
+		>
+			<p>{ translate(
+				'We successfully restored your site back to %s!',
+				{ args: moment( date ).format( 'LLLL' ) }
+			) }</p>
+			<Button primary >
+				{ translate( 'View site' ) }
+			</Button>
+			{ '  ' }
+			<Button onClick={ handleDismiss }>
+				{ translate( 'Got it, thanks' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Thanks, got it!' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).